tests/prague/eip7702_set_code_tx/test_set_code_txs_2.py

```diff
@@ -1689,3 +1691,167 @@ def test_pointer_resets_an_empty_code_account_with_storage(
             Block(txs=[tx_create_suicide_from_pointer]),
         ],
     )
+
+
+@pytest.mark.valid_from("Prague")
+@pytest.mark.parametrize("chain_id", [0, 1])
+def test_chained_delegation_stops_after_one_hop(
```

Comment:
I believe this is the same test case as [pointer_to_pointer](https://github.com/ethereum/execution-spec-tests/blob/eip7702-test-additions/tests/prague/eip7702_set_code_tx/test_set_code_txs_2.py#L93-L134) state test, though the state test does not have a control. I think that's fine. Should I remove this or merge them somehow?